### PR TITLE
CCE-659 unsticks footer

### PR DESCRIPTION
**Pull request by xak**

### CODE REVIEW

**Comment by xak** [comment]
@joseegm The sidebar is fixed now.

**Comment by joseegm** [comment]
@xak tested it again and it looks fine now. Thanks.

File: docs/template/public/less/_side-nav.less [hunk 1]
```diff
@@ -151,26 +151,31 @@ body.has-left-nav .sidebar-nav {
     body.has-left-nav #kss-wrapper {
         padding-left: 265px;
     }
+    .main-content {
+        position: relative; //this is not ideal
```
Comment (like-me1) [inline docs/template/public/less/_side-nav.less]: - delete the comment not related to the functionality explanation 
- probably it's from the past here, but would be great to refactor the file according to Less - cascading. Now it feels like pure CSS (e.g. body.has-left-nav repeated multiple times)

Comment (xak) [inline docs/template/public/less/_side-nav.less]: I removed the comment. The CSS format was already that way. I suspect it was copied from the KSS default styles and modified. Thanks!

**Pull request merged**